src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+const listOfPeople = document.querySelector('ul');
+
+function toNumber(string) {
+  return Number(string.replace(/\D+/g, ''));
```

Comment:
```suggestion
  return Number(string.replace(/\D/g, ''));
```
Без "+" сработает так же